cnxarchive/utils/ident_hash.py

```diff
@@ -40,10 +40,10 @@ def split_ident_hash(ident_hash, split_version=False):
     if HASH_CHAR not in ident_hash:
         ident_hash = '{}@'.format(ident_hash)
     split_value = ident_hash.split(HASH_CHAR)
-    if split_value[0] == '':
-        raise ValueError("Missing values")
 
     try:
+        if split_value[0] == '':
+            raise ValueError("Missing values")
```

Comment:
Should we change the views to also handle `ValueError` and raise `HTTPNotFound`?